lib/core/index.ts

```diff
@@ -54,3 +54,4 @@ export * from './testing';
 
 export * from './material.module';
 export * from './core.module';
+export * from './tooling';
```

Comment:
Nope, please don't do this.

This is exactly what we want to avoid:
```js
import { dotenvConfig } from '@alfresco/adf-core'
```

Instead we want:
```js
import { dotenvConfig } from '@alfresco/adf-core/tooling'
```